### PR TITLE
chore(dependencies): update dependencies

### PR DESCRIPTION
**Pull request by atabel**

### CODE REVIEW

File: .storybook/preview.tsx [hunk 1]
```diff
@@ -16,28 +16,6 @@ import {addons} from '@storybook/addons';
 
 import type {ColorScheme, ThemeConfig} from '../src';
 
-const getUserAgent = () => self.navigator.userAgent || '';
-const isRunningAcceptanceTest = () => getUserAgent().includes('acceptance-test');
-
-const acceptanceStyles = `
```
Comment (atabel) [inline .storybook/preview.tsx:22]: already included in `@telefonica/acceptance-testing`

File: .github/workflows/deploy-master.yml [hunk 2]
```diff
@@ -5,14 +5,9 @@ on:
 jobs:
     deploy:
         runs-on: ubuntu-latest
-        container: telefonica/web-builder:chromium93.0.4577-1.2.4
```
Comment (atabel) [inline .github/workflows/deploy-master.yml:8]: no reason to use the docker in this workflow

File: .github/workflows/deploy-pull-requests.yml [hunk 3]
```diff
@@ -3,14 +3,9 @@ on: [pull_request]
 jobs:
     deploy:
         runs-on: ubuntu-latest
-        container: telefonica/web-builder:chromium93.0.4577-1.2.4
         steps:
             - uses: actions/checkout@v2
 
-            - name: install-vercel
```
Comment (atabel) [inline .github/workflows/deploy-pull-requests.yml:10]: not needed now

File: src/progress-bar.tsx [hunk 4]
```diff
@@ -29,17 +30,29 @@ type Props = {
...
+}) => {
     const classes = useStyles({progressPercent, color});
-
+    const {texts} = useTheme();
+    const defaultLabel = texts.loading;
+    const label = ariaLabelledBy ? undefined : ariaLabel || defaultLabel;
     return (
         <div
             className={classes.barBackground}
+            role="progressbar"
```
Comment (atabel) [inline src/progress-bar.tsx:50]: new version of axe found this issue. We were using `aria-valuenow` without a proper `role`. Also, a label is needed for this component, so I have included `aria-label` and `aria-labelledby` props and default to `text.loading`

File: src/__screenshot_tests__/list-screenshot-test.tsx [hunk 5]
```diff
@@ -112,6 +112,7 @@ test('Rows with only a Title content are centered', async () => {
             subtitle: '',
             headline: '',
             description: '',
+            control: 'chevron',
```
Comment (atabel) [inline src/__screenshot_tests__/list-screenshot-test.tsx:115]: for some reason, the new storybook version doesn't use the chevron as default value for the control arg if not specified and the screenshots were failing

**Review by RubenMorPov — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Pull request merged**